cmd/mattermost/commands/export.go

```diff
@@ -71,9 +71,13 @@ func init() {
 	ScheduleExportCmd.Flags().Int("timeoutSeconds", -1, "The maximum number of seconds to wait for the job to complete before timing out.")
 
 	CsvExportCmd.Flags().Int64("exportFrom", -1, "The timestamp of the earliest post to export, expressed in seconds since the unix epoch.")
+	CsvExportCmd.Flags().Int("batchSize", -1, "The number of posts to export.")
 
 	ActianceExportCmd.Flags().Int64("exportFrom", -1, "The timestamp of the earliest post to export, expressed in seconds since the unix epoch.")
+	ActianceExportCmd.Flags().Int("batchSize", -1, "The number of posts to export.")
```

Comment:
We could add the description, that a negative number disables the limitation. Maybe useful for people trying use it in scripts. 2/5